packages/web/src/components/stat-banner/StatBanner.tsx

```diff
@@ -186,6 +191,13 @@ export const StatBanner = (props: StatsBannerProps) => {
   let buttons = null
   const followButtonRef = useRef<HTMLButtonElement>(null)
   const isManagedAccount = useIsManagedAccount()
+  const chatPermissionStatus = useSelector(getChatPermissionsStatus)
+
+  const [flip, setFlip] = useState(false)
```

Comment:
uh what's this

Reply:
debug stuff, thought I took it out. one sec

Reply:
my hack to see if the sizes match up :)

Reply:
it's gone